lib/config.js

```diff
@@ -333,7 +333,11 @@ function Config(options) {
 
     if (useConfig) {
         debug("Using command line config " + useConfig);
-        this.useSpecificConfig = loadConfig(path.resolve(process.cwd(), useConfig));
+        try {
+            this.useSpecificConfig = loadConfig(path.resolve(process.cwd(), useConfig));
```

Comment:
It seems hard to determine a string indicates whether a name of sharable config or a relative path before trying reading the path.